src/components/editor/AppController.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { browserHistory } from 'react-router';
+import FalcorController from "lib/falcor/FalcorController";
+import { setAppReady } from "lib/falcor/falcorUtils";
+
+export default class AppController extends FalcorController {
+  constructor(props) {
+    super(props);
+    this.safeSetState({
+      defaultValue: this.handleDefaultValue.bind(this)()
+    })
+    
+  }
+  static getFalcorPath() {
+    return;
+  }
+
+  componentDidMount() {
+    super.componentDidMount();
+    setAppReady();
+  }
+
+  handleDefaultValue() {
+    let path = this.props.location.pathname;
+    if (path === '/') {
+      return '/';
+    }
+    else if (path[2] === 'r') {
+      return 'articles';
+    }
+    else {
+      return 'authors';
+    }
+  }
+
+  handleChange(event) {
+    if (event.target.value === '/') {
+      browserHistory.push("/");
```

Comment:
So how would you want me to use the router? Because I googled quite a bit to see how I should do this, and this was what I found as the recommended method. Also see here:
https://github.com/reactjs/react-router/blob/master/docs/guides/NavigatingOutsideOfComponents.md
http://stackoverflow.com/questions/31079081/programmatically-navigate-using-react-router

These were where I was reading anyway. Now that you made me reread them though I see maybe I should either do the thing with making a new component with withRouter (second answer in stackOverflow and recommended by the docs) or this.props.history (but probably not that one as withRouter and browseHistory were the ones that were recommended in the docs)? Just seems more obscure to me, but I must admit when reading I just completely overlooked it, don't remember seeing it. Should I use one of those methods instead?